sdk/core/azure-core/src/main/java/com/azure/core/http/policy/HttpLoggingPolicy.java

```diff
@@ -88,7 +88,8 @@ public HttpLoggingPolicy(HttpLogOptions httpLogOptions) {
                 requestUrl.setQuery(getAllowedQueryString(request.getUrl().getQuery()));
                 logger.info("--> {} {}", request.getHttpMethod(), requestUrl.toUrl());
             } catch (MalformedURLException ex) {
-                return Mono.error(logger.logExceptionAsWarning(new IllegalStateException("Invalid request URL.")));
+                return Mono.error(logger.logExceptionAsWarning(new IllegalStateException("Invalid request URL. URL: "
+                    + request.getUrl())));
```

Comment:
Please view how request sending logging logs the URL, right now this has the potential to log query parameters that should be redacted such a SAS token `sig`.

Reply:
Logging happens after credential policy. Usually it doesn't include sas or other auth. The loggin here especially for response. So it will not expose the secret info

Reply:
Given that is happens after the credential policy wouldn't that mean the SAS would be applied to the request URL?